### PR TITLE
Pinterest and Snapchat Audience Sync

### PR DESCRIPTION
**Pull request by KellieHawks**
# Pull Request/Issue Resolution

Closes #**ISSUE_NUMBER_HERE**

#### Is this change associated with a Braze feature/product release?
- [ ] Yes (**Insert Feature Release Date Here**)
- [ ] No

---

<details>
<summary>✔️ Pull Request Checklist</summary>
<br>

- [ ] Check that you haven't removed any images (replacing an image with an updated one of the same name is fine), as this breaks the French site
- [ ] Check that all links work.
- [ ] Ensure you have completed [our Contributors License Agreement](https://www.braze.com/docs/cla/).
- [ ] Tag @Timothy-Kim and @KellieHawks as a reviewer when your work is **done and ready to be reviewed for merge**. Are you an internal product manager? Reference the internal reviewing chart to tag the appropriate reviewer.
- [ ] Tag others as reviewers as necessary.
- [ ] If you have modified any links, be sure to add redirects to `assets` > `js` > `broken_redirect_list.js`

</details>

<details>
<summary>⭐ Helpful Wiki Shortcuts</summary>
<br>

- [Writing Style Guide](https://docs.google.com/document/d/e/2PACX-1vTluyDFO3ZEV7V6VvhXE4As_hSFwmnFFdU9g6_TrAYTgH1QmbRoEDDdn5GzKAB9vdBbIdyiFdoaJcNk/pub)
- [Image Style Guide](https://docs.google.com/document/d/e/2PACX-1vRJSkwcjmjrTfLDagZccLpOMMyh5NN5SXRZSjz12cRAHbX4OrUmhvCmYpf_p5YB-9r4_jSOQLkicQIH/pub)
- [Styling Test Page](https://www.braze.com/docs/home/styling_test_page/)

</details>

<details>
<summary>❗ ATTN: For PR Reviewers</summary>
<br>

- [ ] Read our [Reviewing a PR page](https://github.com/Appboy/braze-docs/wiki/Reviewing-a-PR) for more on our reviewing suggestions.
- [ ] Read our [Previewing Documentation page](https://github.com/braze-inc/braze-docs/wiki/Previewing-and-Testing-Documentation) to see how to check the deployment.
  - [ ] Preview all changes in the linked Vercel environment by clicking the preview link in the vercel-bot comment in your PR.
</details>

<details>
<summary>❗ ATTN: Internal Reviewing Chart </summary>
<br>
<b>Work at Braze and not sure who to tag for review?</b> <br>Before tagging @timothy-kim or @KellieHawks for a general review, reference the following chart to see if a specific product vertical/reviewer applies to your pull request.
<br><br>
<table>
<tr>
    <td><b>Reviewer</b></td>
    <td><b>Product Vertical</b></td>
  </tr>
  <tr>
    <td>@josh-mccrowell-braze</td>
    <td>Monolith Deployments<br>Quality Infrastructure<br>Platform Infrastructure<br>Datalake<br>SDKs</td>
  </tr>
  <tr>
    <td>@kelliehawks</td>
    <td>Currents<br>Internal Tools<br>Product Partnerships<br>SMS<br>Customer Lifecycle, Identity and Permissions</td>
  </tr>
  <tr>
    <td>@bre-fitzgerald</td>
    <td>Reporting<br>Intelligence<br>User Targeting<br>IAM<br>Channels<br>FIX</td>
  </tr>
  <tr>
    <td>@lydia-xie</td>
    <td>Ingestion<br>Core Objects<br>Core Messaging<br>Messaging and Automation<br>Email (Composition and Infrastructure)</td>
  </tr>
</table>
</details>

### CODE REVIEW

File: _docs/_hidden/private_betas/snapchat_audience_sync.md [hunk 1]
```diff
@@ -0,0 +1,159 @@
+---
+nav_title: Audience Sync to Snapchat
+article_title: Canvas Audience Sync to Snapchat
+description: "This reference article will cover how to use Braze Audience Sync to Facebook, to deliver advertisements based upon behavioral triggers, segmentation, and more."
```
Comment (KellieHawks) [inline _docs/_hidden/private_betas/snapchat_audience_sync.md:4]:
```suggestion
description: "This reference article will cover how to use Braze Audience Sync to Snapchat, to deliver advertisements based upon behavioral triggers, segmentation, and more."
```

File: _docs/_hidden/private_betas/pinterest_audience_sync.md [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+
+Once you have configured your Audience Sync to Pinterest, launch the Canvas! The new audience will be created, and users who flow through the Audience Sync step will be passed into this audience on Pinterest. If your Canvas contains subsequent components, your users will then advance to the next step in their user journey.
+
+You can view the audience on Pinterest by going into your ads manager account and selecting Audiences from the Ads dropdown. From the Audience page, you can see the size of each audience after it reaches ~100.
+
+![Audience details for a given Pinterest audience that includes audience name, audience ID, audience type, audience size.][11]
+
+## User syncing and rate limit considerations
+
+As users reach the Audience Sync Step, Braze will sync these users in near real-time while respecting Pinterest's Marketing API rate limits. In practice, Braze will try to batch and process as many users every 5 seconds before sending these users to Pinterest.
```
Comment (lydia-xie) [inline _docs/_hidden/private_betas/pinterest_audience_sync.md:106]:
```suggestion
As users reach the Audience Sync step, Braze will sync these users in near real-time while respecting Pinterest's Marketing API rate limits. In practice, Braze will try to batch and process as many users every 5 seconds before sending these users to Pinterest.
```

File: _docs/_hidden/private_betas/snapchat_audience_sync.md [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+# Audience Sync to Snapchat
+
+Using the Braze Audience Sync to Snapchat, brands can elect to add user data from their own Braze integration to Snapchat customer lists to deliver advertisements based on behavioral triggers, segmentation, and more. Any criteria you'd normally use to trigger a message (Push, Email, SMS, Webhook, etc.) in a Braze Canvas based on your user data can now be used to trigger an ad to that user in your Snapchat customer lists.
+
+**Common use cases for audience syncing include:**
+
+- Targeting high-value users via multiple channels to drive purchases or engagement
+- Retargeting users who are less responsive to other marketing channels
+- Creating suppression audiences to prevent users from receiving advertisements when they're already loyal consumers of your brand
+- Creating Lookalike Audiences to acquire new users more efficiently
```
Comment (lydia-xie) [inline _docs/_hidden/private_betas/snapchat_audience_sync.md:22]:
```suggestion
- Creating lookalike audiences to acquire new users more efficiently
```
similar audiences?

File: _docs/_hidden/private_betas/snapchat_audience_sync.md [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+
+![Workflow of the previous steps to add a Snapchat Audience component in Canvas Flow.][4]{: style="max-width:60%;"}
+
+![Workflow of the previous steps to add a Snapchat Audience component in Canvas Flow.][5]{: style="max-width:60%;"}
+
+### Step 3: Sync setup
+
+Click on the **Custom Audience** button to open the component editor. Select Snapchat as the desired Audience Sync partner. 
+
+Then select your desired Snapchat ad account. Under the Choose a New or Existing Audience dropdown, type in the name of a new or existing audience.
```
Comment (lydia-xie) [inline _docs/_hidden/private_betas/snapchat_audience_sync.md:72]:
```suggestion
Then select your desired Snapchat ad account. Under the **Choose a New or Existing Audience** dropdown, type in the name of a new or existing audience.
```

File: _docs/_hidden/private_betas/snapchat_audience_sync.md [hunk 5]
```diff
@@ -0,0 +1,159 @@
...
+
+Once you have configured your Audience Sync to Snapchat, simply launch the Canvas! A new audience will be created, and users who flow through the Audience Sync step will be passed into this audience on Snapchat. If your Canvas contains subsequent components, your users will advance to the next step in their user journey.
+
+You can view the audience in Snapchat by going into your ads manager account and selecting **Audiences** from the Assets section of the navigation. From the **Audiences** page, you can see the size of each audience after it reaches ~1,000.
+
+![Audience details for a given Snapchat audience that includes audience name, audience type, audience size, and audience renetion in days.][9]
+
+## User syncing and rate limit considerations
+
+As users reach the Audience Sync Step, Braze will sync these users in near real-time while also respecting Snapchat's API rate limits. In practice, Braze will try to batch and process as many users every 5 seconds before sending these users to Snapchat.
```
Comment (lydia-xie) [inline _docs/_hidden/private_betas/snapchat_audience_sync.md:108]:
```suggestion
As users reach the Audience Sync step, Braze will sync these users in near real-time while also respecting Snapchat's API rate limits. In practice, Braze will try to batch and process as many users every 5 seconds before sending these users to Snapchat.
```

File: _docs/_hidden/private_betas/snapchat_audience_sync.md [hunk 6]
```diff
@@ -0,0 +1,159 @@
...
+
+Snapchat's API rate limit states no more than ten queries per second and 100k users per request. If a Braze customer reaches this rate limit, Braze the Canvas will retry the sync for up to ~13 hours. If the sync is not possible, these users are listed under the Users Errored metric.
```
Comment (lydia-xie) [inline _docs/_hidden/private_betas/snapchat_audience_sync.md:110]:
```suggestion
Snapchat's API rate limit states no more than ten queries per second and 100,000 users per request. If a Braze customer reaches this rate limit, Braze the Canvas will retry the sync for up to ~13 hours. If the sync is not possible, these users are listed under the Users Errored metric.
```

File: _docs/_hidden/private_betas/pinterest_audience_sync.md [hunk 7]
```diff
@@ -0,0 +1,157 @@
...
+
+Pinterest's Segment API rate limit states no more than seven queries per second per user and 1.9k users per request. If a Braze customer reaches this rate limit, Braze the Canvas will retry the sync for up to ~13 hours. If the sync is not possible, these users are listed under the Users Errored metric.
```
Comment (lydia-xie) [inline _docs/_hidden/private_betas/pinterest_audience_sync.md:108]:
```suggestion
Pinterest's Segment API rate limit states no more than seven queries per second per user and 1,900 users per request. If a Braze customer reaches this rate limit, Braze the Canvas will retry the sync for up to ~13 hours. If the sync is not possible, these users are listed under the Users Errored metric.
```

**Review by lydia-xie — CHANGES_REQUESTED** [review]
looks great! left a few suggestions for your review!

File: _docs/_hidden/private_betas/pinterest_audience_sync.md [hunk 8]
```diff
@@ -0,0 +1,157 @@
+---
+nav_title: Audience Sync to Pinterest
+article_title: Canvas Audience Sync to Pinterest
+description: "This reference article will cover how to use Braze Audience Sync to Pinterest, to deliver advertisements based upon behavioral triggers, segmentation, and more."
+page_order: 4
+alias: "/audience_sync_Pinterest/"
```
Comment (KellieHawks) [inline _docs/_hidden/private_betas/pinterest_audience_sync.md:6]:
```suggestion
alias: "/audience_sync_pinterest/"
hidden: true
```

File: _docs/_hidden/private_betas/snapchat_audience_sync.md [hunk 9]
```diff
@@ -0,0 +1,159 @@
+---
+nav_title: Audience Sync to Snapchat
+article_title: Canvas Audience Sync to Snapchat
+description: "This reference article will cover how to use Braze Audience Sync to Snapchat, to deliver advertisements based upon behavioral triggers, segmentation, and more."
+page_order: 4
+alias: "/audience_sync_snapchat/"
+
+Tool:
+  - Canvas
+
```
Comment (KellieHawks) [inline _docs/_hidden/private_betas/snapchat_audience_sync.md:10]:
```suggestion
hidden: true
```

**Review by lydia-xie — APPROVED** [review]

**Pull request merged**